fdbclient/NativeAPI.actor.h

```diff
@@ -165,7 +166,7 @@ struct TransactionInfo {
 };
 
 struct TransactionLogInfo : public ReferenceCounted<TransactionLogInfo>, NonCopyable {
-	enum LoggingLocation { DONT_LOG = 0, TRACE_LOG = 1, DATABASE = 2 };
+	enum LoggingLocation { DONT_LOG = 0, TRACE_LOG = 1, DATABASE = 2, REQ_STATS = 4 };
```

Comment:
I am not seeing where REQ_STATS being added as a LoggingLocation.

Reply:
It's now added with the `TRACK_REQUEST_STATS` transaction option.